protocol.json

```diff
@@ -0,0 +1,1216 @@
+{
+    "type": "object",
```

Comment:
It is something that the generator tool created. We should remove it going forward.